src/Fusion.Summary.Api/Program.cs

```diff
@@ -60,11 +66,14 @@
     {
         opts.ClientId = azureAdClientId ?? throw new InvalidOperationException("Missing AzureAd:ClientId");
         opts.ClientSecret = azureAdClientSecret;
+        opts.CertificateThumbprint = certThumbprint;
     });
 });
 
 builder.Services.AddApplicationInsightsTelemetry();
-builder.Services.AddDbContext<SummaryDbContext>(options => options.UseSqlServer(databaseConnectionString));
+builder.Services.AddSqlDbContext<SummaryDbContext>(databaseConnectionString)
+    .AddSqlTokenProvider<SqlTokenProvider>()
```

Comment:
Could we just have used "AddFusionTokenProvider" directly here?

Reply:
Unsure if I understand. Something like this?

```suggestion
builder.Services.AddTransient<ISqlTokenProvider, SqlTokenProvider>();
builder.Services.AddSqlDbContext<SummaryDbContext>(databaseConnectionString)
    .AddAccessTokenSupport();
```

AddAccessTokenSupport adds SqlAccessTokenDbInterceptor which requires ISqlTokenProvider